resources/leiningen/new/xiana/test/app_name_test.clj

```diff
@@ -2,22 +2,11 @@
   (:require
     [clj-http.client :as http]
     [clojure.test :refer [deftest is use-fixtures]]
-    [com.stuartsierra.component :as component]
     [{{sanitized-name}}]
-    [framework.config.core :as config]))
+    [{{sanitized-name}}-fixture :refer [std-system-fixture]]))
```

Comment:
With more fixtures and test helpers added this ns might be renamed to something like test-utils and eventually moved to the framework itself, for now I am fine